packages/service-clients/end-to-end-tests/odsp-client/package.json

```diff
@@ -30,7 +30,7 @@
 		"test": "npm run test:realsvc:odsp:run",
 		"test:coverage": "c8 npm test",
 		"test:realsvc:odsp": "cross-env npm run test:realsvc:odsp:run",
-		"test:realsvc:odsp:run": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit --timeout 20000",
+		"test:realsvc:odsp:run": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit --timeout 20000 --config src/test/.mocharc.cjs -- --driver=odsp --driverEndpoint=odsp",
```

Comment:
Is there a reason for the `--` (i.e. not passing `--driver=odsp` directly to mocha)?

AFAIK that syntax is mostly just a thing for npm, where generally speaking, command-line arguments to `npm` are not plumbed through to child processes it creates. So e.g. `npm run foo --someFlag` runs the script `foo` and passes `someFlag` to the `npm` process, but whatever code executes as a result of running the `foo` script won't see `--someFlag` in its arguments. However `npm run foo -- --someFlag` is the opposite: the npm process doesn't have `--someFlag` but the script for `foo` will.